### ISSUE TITLE
Item order should be syncronized with Library

### ISSUE THREAD

**Issue by scienmind** [MatMoul/quickdial-webext#5]
Changing item's order in library is not reflected in Quick Dial item's layout, and vice versa, changing Quick Dials order, does not update the order in Library.

FF 56.0

**Comment by MatMoul** [comment]
In a first time, I got inspired with fast dial that have the same operation.
The sync function need some work now but syncing index is very difficult with empty tab.
However, I will consider a solution.

**Comment by scienmind** [comment]
Thanks for considering the requests.

---

Off topic:
Your SW seem to be very promising: it's fast, responsive and have the basics: folders, back-fwd navigation, some basic customization. 

Coming from the original Opera Presto's speed dial, it was quite a challenge to find a replacement for this functionality for other browsers. They're all either slow and bloated, or can't use bookmarks as a base which makes it hard to sync stuff across devices and export/import, or have some basic flaws like not supporting sub-folders or navigating back from a mis-clicked folder.

Eventually I found this one for chromium-based browsers: https://github.com/j3nda/speed-dial-chrome
It does everything quite right. 

Now I'm thinking of switching to FF for 10^10th time, and with FF 57 closing in, seems like it'll finally be a decent browser performance-wise. Unfortunately, Simple Speed Dial doesn't seem to be in active dev. since 2014, so porting to FF is probably not on the horizon.

Long story short, I'm looking for a decent speed dial, and seems like your Quick Dial is on the right track.
Keep up the good work! 👍

**Comment by scienmind** [comment]
What is your intent of "question" label? A question to the OP? Or "the legitimacy of the issue is in question for now"?

**Comment by MatMoul** [comment]
The project has only one week, I don't maked custom tag and I use default...
Enhancement is for feature accepted and question is for discution or future enhancement.

**Comment by fernandosantucci** [comment]
You can use the field "Description" of Firefox Favorites to store a JSON with image url, cell position (row, column), then Quick Dial will can use the Firefox Sync and this configurations not will miss never more.

**Comment by MatMoul** [comment]
@fernandosantucci 
This is an option...
I've tested too an additional solution that use the separators in bookmarks to represent a empty cell in Quick Dial.
But these two solution have a side effect on bookmarks.
